macOS/InfoGold.plist

```diff
@@ -94,6 +94,10 @@
 	<true/>
 	<key>NSHumanReadableCopyright</key>
 	<string></string>
+	<key>LSApplicationCategory</key>
+	<string>public.app-category.games</string>
```

Comment:
This value should be used for `LSApplicationCategoryType` instead of `LSApplicationCategory` isn't? https://developer.apple.com/documentation/bundleresources/information_property_list/lsapplicationcategorytype

PS: `LSApplicationCategory` key might not even exist

Reply:
Oops, that's a typo by me indeed. I'll fix it.